modules/ROOT/partials/konflux-key-features.adoc

```diff
@@ -1,27 +1,52 @@
-* *Robust Security*
+=== CI/CD Automation
 
-+
-Enhance the security of your software supply chain. {ProductName} offers unparalleled transparency into your software’s composition, helping you address vulnerabilities before they become issues.
+{ProductName} simplifies your DevOps processes through:
 
-* *Customizable Compliance*
+* *Automated Pipelines*: {ProductName} provides default pipeline configurations for building, testing, and deploying production software. You can trigger these pipelines via Git events, such as when team members accept push or pull requests. {ProductName} posts the results back to your source control management system.
 
-+
-Achieve the highest security standards and customize your development process to meet specific project requirements. {ProductName}'s adaptable platform ensures compliance without sacrificing flexibility.
+* *Integration with GitHub*: {ProductName} allows you to integrate with OpenShift pipelines using a GitHub app. This integration enables you to run Texton PipelineRuns based on GitHub events, which you can define in your repository's `.tekton` folder.
 
-* *Efficient Deployment*
+* *Red Hat Subscription Support*: {ProductName} supports activation keys, allowing you to access Red Hat software using tools like `buildah` and `prefetch-dependencies`, which simplifies dependency management in your software supply chain.
 
-+
-Build once, deploy anywhere. {ProductName} optimizes your release process, allowing for multiple deployments without redundancy and significantly reducing time-to-market.
+=== Collaboration
+
+{ProductName} encourages you to collaborate with its open-source community through resources such as the {ProductName} community site and eBooks on modern DevOps practices.
+
+=== Customization and Flexibility
+
+{ProductName} allows you to customize your project to meet your specific requirements, maintain high security standards, and adapt to various enterprise needs. {ProductName}'s adaptable platform ensures compliance without sacrificing flexibility.
+
+=== Open Source
+
+{ProductName} is distributed under the Apache 2.0 license and is built on top of Kubernetes. You can set {ProductName} up locally or as a cloud-native platform for your development teams. {ProductName} utilizes standard command-line tools, such as `kubectl` and `oc`, for managing resources.
```

Comment:
if you mention k8s then only kubectl should be mentioned, or add Kubernetes (Openshift) and kubectl (oc)

Reply:
in previous paragraph, you were referring only to Openshift instead of k8s, we should stay consistent IMO and pick only one or both